test/conf/pint/pint-config.hcl

```diff
@@ -0,0 +1,57 @@
+rule {
+  # Disallow spaces in label/annotation keys, they're only allowed in values.
+  reject ".* +.*" {
+    label_keys      = true
+    annotation_keys = true
+  }
+
+  # Disallow URLs in labels, they should go to annotations.
+  reject "https?://.+" {
+    label_keys   = true
+    label_values = true
+  }
+
+  # Ensure that all aggregations are preserving "job" label.
+  aggregate ".+" {
+    severity = "bug"
+    keep     = ["cluster_id", "installation", "pipeline", "provider"]
+  }
+}
+
+rule {
+  # This block will apply to all alerting rules.
+  match {
+    kind = "alerting"
+  }
+
+  # Each alert must have a 'summary' annotation on every alert.
+  annotation "description" {
```

Comment:
Fix the comment

Reply:
Oops, I fixed the comments @QuentinBisson :+1:

Reply:
Thank you